jobserver/views/errors.py

```diff
@@ -0,0 +1,66 @@
+from django.template.response import TemplateResponse
+
+
+def bad_request(request, exception=None):
+    return TemplateResponse(
+        request,
+        "error.html",
+        status=400,
+        context={
+            "error_code": "400",
+            "error_name": "Bad request",
+            "error_message": "An error has occurred displaying this page.",
+        },
+    )
+
+
+def csrf_failure(request, reason=""):
```

Comment:
I couldn't see this used anywhere in the code, so went digging in Interactive to see if I could find out how it worked. That has this entry in the settings.py file

`CSRF_FAILURE_VIEW = "interactive.views.csrf_failure"`

Presumably, we need something similar here or in errors.py?